### PR TITLE
Keyboard/miniashen40

### PR DESCRIPTION
**Pull request by jfescobar18**
## Description

New keyboard added, already tested and working with physical device, and functional prototype

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/mechanickeys/miniashen40/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  - Mini macro cluster
+  - Full assembly with only through hole components
+
+* Keyboard Maintainer: [jfescobar18](https://github.com/jfescobar18)
+* Hardware Supported: Mini Ashen 40 PCB and Case
+* Hardware Availability: [MechanicKeys](https://www.facebook.com/MechanicKeys-104963764775280)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make miniashen40/:default
```
Comment (zvecr) [inline keyboards/mechanickeys/miniashen40/README.md:20]:
```suggestion
    make mechanickeys/miniashen40:default
```

File: keyboards/mechanickeys/miniashen40/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+Flashing example for this keyboard:
+
+    make miniashen40:default:flash
```
Comment (zvecr) [inline keyboards/mechanickeys/miniashen40/README.md:24]:
```suggestion
    make mechanickeys/miniashen40:default:flash
```

File: keyboards/mechanickeys/miniashen40/rules.mk [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
```
Comment (zvecr) [inline keyboards/mechanickeys/miniashen40/rules.mk:31]:
```suggestion
# MCU name
MCU = atmega328p

# Bootloader selection
BOOTLOADER = USBasp

# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = no            # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/mechanickeys/miniashen40/README.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+A 40% keyboard with some extras
+  - Arrows
+  - Mini macro cluster
+  - Full assembly with only through hole components
+
+* Keyboard Maintainer: [jfescobar18](https://github.com/jfescobar18)
+* Hardware Supported: Mini Ashen 40 PCB and Case
+* Hardware Availability: [MechanicKeys](https://www.facebook.com/MechanicKeys-104963764775280)
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/mechanickeys/miniashen40/README.md:18]: Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/mechanickeys/miniashen40/info.json [hunk 5]
```diff
@@ -0,0 +1,17 @@
+{
+  "keyboard_name": "Mini Ashen 40",
+  "url": "",
+  "maintainer": "qmk",
+  "width": 14.25,
+  "height": 4.25,
```
Comment (fauxpark) [inline keyboards/mechanickeys/miniashen40/info.json:6]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by jfescobar18** [comment]
Any updated on this?

**Comment by jfescobar18** [comment]
@drashna  @fauxpark @zvecr 

Any update??

**Comment by drashna** [comment]
Sorry about that.

And thanks!

**Pull request merged**